src/app/app.constants.ts

```diff
@@ -0,0 +1,26 @@
+const FooterData =  {
+  heading: 'Help',
```

Comment:
Is there a reason why this isn't in the config.json, and exposed through the AppConfigService? 

It's OK for now, but please add a backlog task to refactor this into the config service.

Reply:
This is static content which never changes. This is why it is in constants file.

Reply:
i agree with Boris - don't think this needs to be configurable, so no need to be in config file unless is decided to be configurable by the business